nimrod/report/report_analysis.py

```diff
@@ -0,0 +1,170 @@
+
+    def methods_report_comparison(self, path_suite_original, path_suite_modified):
+        methods_data = {}  # A dictionary in which the method name is the key and the value returned is a list with
+        # the number of times that the method was called for original randoop and randoop--
+
+        try:
+            methods_report_file = open(path_suite_original + "/methods_report.csv")
+            methods_report = methods_report_file.read()
+            methods_report_file.close()
+
+            lines = methods_report.split("\n")
+            for line in lines[1:-1]:  # Exclude the first one because contains the headers and the last one because it is a empty line
+                cells = re.split("(?<=\"),", line)
+                method_map = {cells[0]: [cells[1], 0]}
+                methods_data.update(method_map)
+        except:
+            print("Error on methods_report.csv in " + str(path_suite_original))
+            return {}
+
+        try:
+            methods_report_file = open(path_suite_modified + "/methods_report.csv")
+            methods_report = methods_report_file.read()
+            methods_report_file.close()
+
+            lines = methods_report.split("\n")
+            for line in lines[1:-1]:  # Exclude the first one because contains the headers and the last one because it is a empty line
+                cells = re.split("(?<=\"),", line)
+                if cells[0] in methods_data:  # If the method signature is already in the dictionary, just change the value in the list
+                    methods_data.get(cells[0])[1] = cells[1]
+                else:
+                    method_map = {cells[0]: [0, cells[1]]}
+                    methods_data.update(method_map)
+        except:
+            print("Error on methods_report.csv in " + str(path_suite_modified))
+            return {}
+
+        return methods_data
+
+    def objects_report_comparison(self, path_suite_original, path_suite_modified):
+        objects_data = {}  # A dictionary the class of the object is the key and the value returned is a list with
+        # the number of times that a objects from the class were created and how many of them are different from each
+        # other, in that order, from randoop and randoop--
+
+        try:
+            objects_report_file = open(path_suite_original + "/objects_report.csv")
+            objects_report = objects_report_file.read()
+            objects_report_file.close()
+
+            lines = objects_report.split("\n")
+            for line in lines[1:-1]:  # Exclude the first one because contains the headers and the last one because it is a empty line
+                cells = line.split(",")
+                object_map = {cells[0]: [cells[1], cells[2], 0, 0]}
+                objects_data.update(object_map)
+        except:
+            print("Error on objects_report.csv in " + str(path_suite_original))
+            return {}
+
+        try:
```

Comment:
The previous comment I made can be applied here.
There is duplicated code in this class that could be eliminated by extracting these duplicated lines as a new method.